airflow/api_connexion/schemas/dag_schema.py

```diff
@@ -50,6 +50,7 @@ class Meta:
         model = DagModel
 
     dag_id = auto_field(dump_only=True)
+    dag_display_name = fields.Method("get_dag_display_name", dump_only=True)
```

Comment:
Can this not just be an `auto_field` or `fields.String`? There’s no special logic needed here. Same for `task_display_name` below.

Reply:
Thanks for the feedback. `auto_field()`is not working because it is a hybrid property and I did not want to expose the bare DB field.
But actually via `task_display_name = fields.String(attribute="task_display_name")` it is working.